src/Service/Accounts/AccountsService.php

```diff
@@ -368,12 +371,45 @@ protected function getResponseErrorMsg(Response $response, $defaultMessage = '')
     {
         $msg = $defaultMessage;
         $body = $response->body;
-        if (isset($body['error']) &&
+        if (
+            isset($body['error']) &&
             isset($body['error_description'])
         ) {
             $msg = $body['error'] . ': ' . $body['error_description'];
         }
 
         return $response->statusCode . ' - ' . $msg;
     }
+
+    /**
+     * @param string $cloudShopId
+     * @param string $shopToken
+     *
+     * @return IdentityCreated
+     *
+     * @throws AccountsException
+     */
+    public function migrateShopIdentity($cloudShopId, $shopToken, ShopUrl $shopUrl)
```

Comment:
public function should be defined before protected ones (at the end of the class)